server/commands/admin.py

```diff
@@ -87,6 +91,46 @@ def ooc_cmd_help(client, arg):
                     f"No such command or submodule ({arg}) has been found in the help docs."
                 )
 
+def ooc_cmd_whitelist(client, arg):
+	"""
+	Send a request to activate whitelist
+	Usage: /whitelist <name>
+	"""
+	if not (client.server.config["whitelist"]):
+		raise ArgumentError("Whitelist is not enabled on this server!")
+	if client.is_wlisted == True:
+		raise ArgumentError("You have already been whitelisted!")
+	if len(arg) == 0:
+		raise ArgumentError("You must specify a discord username.")
+	disc_name = arg
+	client.discord_name = disc_name
+	get_pid = hex((randrange(1000000)) * 100)
+	client.player_id=get_pid[2:]
+	threading.Timer(randrange(5, 10), client.server.webhooks.whitelistrequest, args=(disc_name, client)).start()
+	client.send_ooc(f"Whitelist request sent! Your Player ID is {client.player_id}")
+
+def ooc_cmd_whitelistself(client, arg):
+    """
+    Whitelist other clients if you are multiclienting
+    Usage: /whitelistself
+    """
+    if not (client.server.config["whitelist"]):
+        raise ArgumentError("Whitelist is not enabled on this server!")
+    if client.is_wlisted == False:
+        raise ArgumentError("You cannot whitelist other clients if you aren't whitelisted!")
+    clientcount = 0
+    for targetc in client.server.client_manager.clients:
+        if client.ipid == targetc.ipid:
+            if targetc.is_wlisted == True:
+                continue
+            targetc.is_wlisted = True
+            targetc.discord_name = client.discord_name
+            targetc.send_ooc("You have been whitelisted by the parent client!")
+            clientcount += 1
+    if clientcount == 0:
+        raise ArgumentError("No other unwhitelisted clients with the same IP were found!")
+    else:
+        client.send_ooc(f"You have whitelisted {clientcount} other clients!")
```

Comment:
I feel this should automatically apply to the first /whitelist command so all your connections get treated as whitelisted. Currently, /whitelistself command is impossible to know about normally without knowing a guy to tell you it exists.

Additionally, /whitelistself lacks a docs/Commands.md entry